src/Saritasa.Tools.SourceGenerator/Constants.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace Saritasa.Tools.SourceGenerator;
+
+/// <summary>
+/// Solution constants.
+/// </summary>
+internal class Constants
+{
+    /// <summary>
+    /// Do not notify attribute name.
+    /// </summary>
+    public static string DoNotNotifyAttributeName = "DoNotNotifyAttribute";
+
+    /// <summary>
+    /// Also notify attribute name.
+    /// </summary>
+    public static string AlsoNotifyAttributeName = "AlsoNotifyAttribute";
+
+    /// <summary>
+    /// Autogenerated attributes.
+    /// </summary>
+    public static readonly string Attributes = """
```

Comment:
Cuz we can't generate this code by other possibilities. This attributes will be generated automatically, it's allow developers to use this attributes within other assemblies.

Reply:
Because the `Roslyn Source Generator` is **additive** only. The consumer project cannot refer to the code of `source generator` project (the only way is generate the attributes in compile time for consumer project).
That means consumer refer to `Source Generator` project, BUT canno't access the source code of this generator project.

https://github.com/dotnet/roslyn/blob/main/docs/features/source-generators.cookbook.md#proposal

Reply:
Ok, can we have the comment about this in the code?